src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -662,6 +662,36 @@ internal static void SetCurrentThreadUiCulture(CultureInfo uiCultureInfo)
         #region Misc
 
         /// <summary>
+        /// Facade for Convert.ToBase64String(bytes, Base64FormattingOptions.InsertLineBreaks)
+        /// Inserts line breaks after every 76 characters in the string representation.
+        /// </summary>
+        internal static string ToBase64StringWithLineBreaks(byte[] bytes)
```

Comment:
Isn't `[convert]::ToBase64String($test.ToCharArray(), 0, 100,1)` your workaround in powershell? You can do this in powershell because the API `ToBase64String(byte[] inArray, int offset, int length, System.Base64FormattingOptions)` is exposed in runtime assembly and thus powershell is able to call it using reflection. However, the contract assembly doesn't contain this API. C# code are compiled against the contract assembly, and there this won't work in C#.